cypress/e2e/3-default-owner.cy.ts

```diff
@@ -1,9 +1,115 @@
-describe('Authentication', () => {
-	it('should skip owner setup', () => {
-		cy.skipSetup();
+import { randFirstName, randLastName } from '@ngneat/falso';
+import { DEFAULT_USER_EMAIL, DEFAULT_USER_PASSWORD } from '../constants';
+import { SettingsUsersPage, SignupPage, WorkflowsPage, WorkflowPage, CredentialsPage, CredentialsModal, MessageBox } from '../pages';
+
+import { MainSidebar, SettingsSidebar } from "../pages/sidebar";
+
+const mainSidebar = new MainSidebar();
+const settingsSidebar = new SettingsSidebar();
+
+const workflowsPage = new WorkflowsPage();
+const signupPage = new SignupPage();
+const workflowPage = new WorkflowPage();
+
+const credentialsPage = new CredentialsPage();
+const credentialsModal = new CredentialsModal();
+
+const settingsUsersPage = new SettingsUsersPage();
+
+const messageBox = new MessageBox();
+
+const username = DEFAULT_USER_EMAIL;
+const password = DEFAULT_USER_PASSWORD;
+const firstName = randFirstName();
+const lastName = randLastName();
+
+describe('Default owner', () => {
+	// todo test should redirect to setup if have not skipped
+
+	beforeEach(() => {
+		cy.task('db:reset');
 	});
 
-	// todo test for adding workflow
-	// todo test for setting up UM again through settings
-	// todo test that workflows migrated successfully
+	it('should be able to use n8n without user management and setup UM', () => {
+		describe('should skip owner setup', () => {
+			cy.skipSetup();
+			cy.url().should('include', workflowsPage.url);
+		});
+
+		describe('should be able to create workflows', () => {
```

Comment:
Do we need this? Wouldn't this already be tested by the `workflows` spec? Or perhaps we can move the missing bits to this spec to also test reload: 
```
cy.reload();

workflowPage.getters.workflowNameInput().should('contain.value', 'Test workflow');
```

Reply:
this is in the special case of no user management... we have run into bugs before we forget users don't have this setup... that's why I am covering it here again. and the reload.. don't want the user to be redirected to sign in or anything